notes/stv/20200429-spark-tips-ML.txt

```diff
@@ -0,0 +1,170 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2020, ROE (http://www.roe.ac.uk/)
+#
+#     This information is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This information is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#  
+#     You should have received a copy of the GNU General Public License
+#     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#   </meta:licence>
+# </meta:header>
+#
+
+
+
+# ---------------------------#
+#      ML Spark Tips         #
+# ---------------------------#
+
+# The following assumes a dataset that looks like this:
+
+root
+ |-- households: string (nullable = true)
+ |-- housingMedianAge: string (nullable = true)
+ |-- latitude: string (nullable = true)
+ |-- longitude: string (nullable = true)
+ |-- medianHouseValue: string (nullable = true)
+ |-- medianIncome: string (nullable = true)
+ |-- population: string (nullable = true)
+ |-- totalBedRooms: string (nullable = true)
+ |-- totalRooms: string (nullable = true)
+
```

Comment:
Is this this data set ?
https://www.datacamp.com/community/tutorials/apache-spark-tutorial-machine-learning#loading-in-your-data
https://www.dcc.fc.up.pt/~ltorgo/Regression/cal_housing.html